Tests/AppTests/SaltHashTests.swift

```diff
@@ -4,19 +4,5 @@ import Crypto
 
 final class SaltHashTests: XCTestCase {
     func testSalt() async throws {
```

Comment:
Remove if unused

Reply:
need to have a single test or else building the test target will fail. kept for future use.